persistence/sql/persister_verification.go

```diff
@@ -33,6 +33,8 @@ func (p Persister) GetVerificationFlow(ctx context.Context, id uuid.UUID) (*veri
 		return nil, sqlcon.HandleError(err)
 	}
 
+	r.SetReturnTo()
```

Comment:
Implement `db.Scan` instead which sets this, then this field will always be set when loaded from the DB

Reply:
It looks like this one may be a little tricky - I've tried making ReturnTo a new type that just wraps string, and implemented the Scan method on the new type (along with setting the db tag to 'request_url' and the rw tag to 'r'), but it doesn't seem to be picking this up when I re-run the tests with this change (presumably as that column is already referenced?)

Is this what you were thinking @aeneasr or did you have another approach in mind?

Reply:
Sorry yeah, it's indeed not possible to use a custom `db.Scan` because we this won't work on the outer struct. I thought sql uses the same concept as json but apparently not so!